### DISCUSSION TITLE
账号不知为何被封禁

### DISCUSSION THREAD

**Discussion by Dvgd123** [BurnHair/docs#133] [技术支持]
{
  "error": {
    "message": "用户3498已被封禁： (request id: 202406281501082030275309BJJsxze)",
    "type": "one_api_error"
  }
}

您好,我的账户不知道为何会被封禁;
不存在重复注册的行为,不知道是否是因为Android,PC多端使用以及有时挂梯子访问, 引起的ip不同导致账号被封禁了

**Comment by BennyThink** [comment]
和IP变化、设备使用没关系的。

封禁的原因是你与其他多位用户 被检测到了重复注册。你确定自己没有重复注册吗？

**Reply by Dvgd123** [reply]
> 和IP变化、设备使用没关系的。
> 
> 封禁的原因是你与其他多位用户 被检测到了重复注册。你确定自己没有重复注册吗？

我的令牌曾分享过给我的朋友, 他在体验后注册了自己的账户, 是否和这个原因有关

**Reply by BennyThink** [reply]
有这个可能。已经解封了。

**Reply by Dvgd123** [reply]
> 有这个可能。已经解封了。

好的, 谢谢您的帮助
